.stylelintrc

```diff
@@ -1,28 +1,23 @@
 {
   "plugins": [
     "stylelint-selector-bem-pattern",
-    "stylelint-scss"
+    "stylelint-scss",
```

Comment:
I think `stylelint-scss` is unnecessary as `stylelint-config-recommended-scss`  includes `stylelint-scss` https://github.com/stylelint-scss/stylelint-config-recommended-scss/blob/master/index.js

Reply:
removed